src/main/java/org/folio/circulation/services/feefine/FeeRefundProcessor.java

```diff
@@ -21,6 +22,11 @@
 import org.joda.time.DateTime;
 
 public class FeeRefundProcessor implements AccountRefundProcessor {
+  private static final String PAYMENT_CREDIT_TRANSACTION_INFO = "Refund to patron";
+  private static final String PAYMENT_REFUND_TRANSACTION_INFO = "Refunded to patron";
+  private static final String TRANSFER_CREDIT_TRANSACTION_INFO = "Refund to %s";
+  private static final String TRANSFER_REFUND_TRANSACTION_INFO = "Refunded to %s";
```

Comment:
Given these constants only appear to be used in a single place each, could they be moved to a closer scope to where they are used?

Reply:
Moved.